docker-compose.yaml

```diff
@@ -2,5 +2,7 @@ version: '2'
 services:
   app:
     image: swisstopo/service-alti:local
+    environment:
+      - DTM_BASE_PATH=${DTM_BASE_PATH}
```

Comment:
The `=${DTM_BASE_PATH}` would not be required here, it would also worked without it.
```yaml
services:
  app:
    image: swisstopo/service-alti:local
    environment:
      - DTM_BASE_PATH
```